lua/telescope/_extensions/zk.lua

```diff
@@ -9,7 +9,7 @@ end
 ---@param opts? table additional options for zk, telescope options, all optional and in one table
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zktaglist
 local function show_tags(opts)
-  zk.edit_from_tags(opts, { picker = "telescope", telescope = opts })
+  require("zk.commands").get("ZkTags")(opts, { picker = "telescope", telescope = opts })
```

Comment:
This doesn't work, ZkTags only takes one argument and therefore ignores picker="telescope". I've removed it now.